docs/source/user/fast.farm/FFarmTheory.rst

```diff
@@ -1291,6 +1292,135 @@ the solution of the momentum equation),
 :math:`V_{r_{n_p,n_r-1}}^\text{Wake}\left[ n+1 \right]` for
 :math:`1\le n_r\le N_r-1`. [10]_
 
+
+
+.. _FF:WAT:
+
+Wake-Added Turbulence (WAT)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Wake-added turbulence is the additional small-scale turbulence generated
+from the turbulent mixing in the wake. 
+It is modeled by scaling up a background (undisturbed) turbulence.
+
+
+The theory for WAT will be presented in more details in the future as part of a journal publication.
+
+
+The basic principle for the wake-added turbulence is illustrated in :numref:`FF:WATSketch`. 
+
+.. figure:: Pictures/FFWakeAddedTurbBoxCoord.svg
+   :alt: Wake Added turbulence
+   :name: FF:WATSketch
+   :width: 100%
+   :align: center
+
+   Wake-added turbulence
+
+A scaling factor is computed at each wake plane, it is multiplied with a unit turbulence box
+and added to the quasi steady wake to form the final wake with wake-added turbulence. 
+In this implementation, the scaling factors are computed in the meandering
+frame, but assembled with the “global” unit turbulence box in the global
+frame. More details follow.
+
+
+**Scaling factor**
+
+The scaling factor, expressed in terms of the wake deficit :math:`V_x^{Wake}`, is determined at each wake plane as:
+
+.. math::
+
+   \begin{aligned}
+      k_{} (x,y,z) = 
+          \frac{k_\text{def} }{\overline{U}} \left| V_x^{Wake}(x,y,z) \right|
+        + \frac{k_\text{grad}D}{2\overline{U}} \left[\left|{\frac{\partial {V_x^{Wake}(x,y,z)}}{\partial r}}\right| +  \left|{\frac{1}{r}\frac{\partial {V_x^{Wake}(x,y,z)}}{\partial \theta}}\right|  \right]  
+        \end{aligned}
+
+where :math:`D` is a reference diameter, and :math:`\bar{U}` is the mean velocity taken as the filtered velocity at the turbine location normal to the rotor disk.
+The coordinates :math:`x,y,z` and :math:`r,\theta` are taken in the meandering frame of reference.
+The parameters :math:`k_\text{def}` and :math:`k_\text{grad}` are tuning constant of the model. They will be calibrated in future studies.
+
+
+
+**Unit turbulence boxes** 
+
+The 3 turbulence Mann boxes are stored as a 4D
+array :math:`\boldsymbol{u}_\text{unit}` of dimension
+:math:`(3,n_x, n_y, n_z)`.
+The turbulence boxes used for the WAT are isotropic turbulence boxes
+with unit standard deviation, generated using the Mann
+model :cite:`ff-Mann1994`.
+To generate a box with unit standard deviation, the dissipation rate is set to:
+
+.. math::
+
+      \alpha\epsilon^{2/3}\approx \frac{1}{0.688 L^{2/3}} 
+
+We have found that there is no dependency on the length scale. We nevertheless recommend to set it to the rotor diameter if the users generate their own boxes.
+
+
+**Predefined boxes**
+
+A recommended practice for the high-resolution domain of FAST.Farm is to chose a grid
+spacing equal to the maximum chord of the blade. 
```

Comment:
This is no longer true.  See: https://github.com/OpenFAST/openfast/issues/1822.  Will this impact WAT?

Reply:
I think we can keep this for WAT as the chord max is a good recommendation for the loads. But we could use D/15 as well (about double)

Reply:
Never mind.  My prior comment was incorrect.  The D/15 guidance is for dr, not DS_High.  I agree that setting DS_High equal to max chord is good guidance.